src/erc165/decoders/index.ts

```diff
@@ -0,0 +1,17 @@
+import { createAbiDecoder, DecoderDefinition, extractParamValue } from '../../core/services/decoder';
+
+type Erc165LogBindings = {};
+
+type Erc165TxBindings = {};
+
+class Erc165InterfaceDecoder implements DecoderDefinition<Erc165TxBindings, Erc165LogBindings> {
```

Comment:
Why is this needed?

Reply:
you're right. Part of the old implementation before I have started  to use the erc165Service. Deleted.